donate.html

```diff
@@ -114,7 +114,7 @@ <h2>
         </span>
       </h2>
         <img src="img/technica-logo.png" width="270" height="60">
-        <div class="b1"> <button>Donate</button> </div>
+        <div class="b1"> <button class="donate-button">Donate</button> </div>
```

Comment:
could you change this to be a link instead of a button like how you did for consistency?